packages/adapter-supabase/src/index.ts

```
@@ -334,101 +495,187 @@
-        }));
-    }
 
-    async getMemoryById(memoryId: UUID): Promise<Memory | null> {
-        const { data, error } = await this.supabase
-            .from("memories")
-            .select("*")
-            .eq("id", memoryId)
-            .single();
+        if (!embedding.every(n => typeof n === 'number' && !isNaN(n))) {
+            elizaLogger.error("Invalid embedding values - must all be numbers", {
+                invalidValues: embedding.filter(n => typeof n !== 'number' || isNaN(n))
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)